lib/doh.c

```diff
@@ -182,6 +186,33 @@ doh_write_cb(const void *contents, size_t size, size_t nmemb, void *userp)
   return realsize;
 }
 
+#ifdef USE_HTTPSRR
+static void local_print_buf(struct Curl_easy *data,
+                            unsigned char *buf, size_t len)
```

Comment:
Maybe this should also be within `#ifdef CURLDEBUG` ?

Reply:
fair point, did that